com.unity.shadergraph/Editor/Data/Nodes/Procedural/Noise/SimpleNoiseNode.cs

```diff
@@ -48,7 +48,11 @@ public override void GenerateNodeFunction(FunctionRegistry registry, GenerationM
             registry.ProvideFunction("Unity_SimpleNoise_RandomValue_$precision", s => s.Append(@"
 inline $precision Unity_SimpleNoise_RandomValue_$precision ($precision2 uv)
 {
-    return frac(sin(dot(uv, $precision2(12.9898, 78.233)))*43758.5453);
+    $precision angle = dot(uv, $precision2(12.9898, 78.233));
+    #if defined(SHADER_API_MOBILE) && (defined(SHADER_API_GLES) || defined(SHADER_API_GLES3) || defined(SHADER_API_VULKAN))
+        angle = fmod(angle, TWO_PI); // Avoid large inputs to sin()
```

Comment:
Might want to add the "why" to the comment here:  something about Mali not liking it...
So when someone comes along and wonders if they can remove this, they know what to test...

Reply:
That would be good, even better would be to also add a test to ensure we don't re-break it!

Reply:
Added a comment